lang/strings_english.txt

```diff
@@ -178,6 +178,7 @@ $s_opensearch_text_description = '%s: full-text search';
 $s_select_option = '(select)';
 
 # bug_actiongroup_page.php : mass treatment
+$s_bug_actiongroup_result = 'The following issues could not be processed';
```

Comment:
This change is not related to loading status_config.php CSS

Reply:
You're right it belongs to another branch I was working on in parallel, it was committed by mistake.